.github/workflows/main.yml

```diff
@@ -352,6 +352,15 @@ jobs:
           git -C libultraship checkout nx/main-nx
           wget https://github.com/Kitware/CMake/releases/download/v3.28.3/cmake-3.28.3-linux-x86_64.sh -O /tmp/cmake.sh
           sudo sh /tmp/cmake.sh --prefix=/usr/local/ --exclude-subdir
+          wget https://github.com/leethomason/tinyxml2/archive/refs/tags/10.0.0.tar.gz
+          tar -xzf 10.0.0.tar.gz
+          cd tinyxml2-10.0.0
+          mkdir build
+          cd build
+          cmake -H.. -B. -DCMAKE_TOOLCHAIN_FILE=/opt/devkitpro/cmake/Switch.cmake
+          make
+          sudo make install
+          cd ../..
```

Comment:
i was getting an error:
```
'class tinyxml2::XMLElement' has no member named 'InsertNewChildElement'
```

since that method does exist in the docs, i assumed it was a version issue. i saw "Install latest tinyxml2" steps for some of the other platforms. this is just a modified version of those.